app/models/translation.rb

```diff
@@ -18,6 +32,13 @@ def time_updated
     self.time_last_updated
   end
 
+  def update_status_to_completed
+    unless self.status == @@COMPLETE_WITH_PRIORITY_STATUS
```

Comment:
I don't think you've defined `@@COMPLETE_WITH_PRIORITY_STATUS`. Better to just abstract completion as a method and check with that.